package.json

```diff
@@ -20,37 +20,37 @@
   },
   "scripts": {
     "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
-    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test": "nyc --reporter=text mocha --recursive 'test/**/*.js' --reporter spec --timeout 5000 --ui bdd --exit",
     "test:watch": "npm run test -- -w ./lib",
     "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
-    "lint:md": "remark 'README.md' documentation",
-    "lint:text": "textlint 'README.md' 'documentation/*.md' 'documentation/**/*.md'",
-    "prettier": "prettier --config .prettierrc.json --write ./**/**/*.js **/*.js *.js",
-    "prettier:text": "prettier 'README.md' 'documentation/*.md' 'documentation/**/*.md' --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
-    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "lint:md": "remark  -f '*.md' 'documentation/*.md'",
+    "lint:text": "textlint '*.md' 'documentation/*.md'",
+    "prettier": "prettier --config .prettierrc.json --write '**/**/**/**/*.js' '**/**/**/*.js' '**/**/*.js' '**/*.js' '*.js'",
+    "prettier:text": "prettier 'README.md' 'doc/*.md' 'doc/**/*.md' --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
+    "test:coverage": "nyc --reporter=lcov mocha -- --recursive 'test/**/*.js' --reporter spec --timeout 5000 --exit",
     "test:coveralls": "npm run test:coverage && cat ./coverage/lcov.info | coveralls && rm -rf ./coverage",
     "watch": "watch 'npm test && npm run lint' ./lib ./test"
   },
   "devDependencies": {
     "chai": "~4.1.2",
     "coveralls": "~3.0.2",
     "husky": "~1.1.0",
-    "istanbul": "~0.4.5",
+    "nyc": "~15.1.0",
     "jshint": "~2.9.6",
     "lint-staged": "~7.3.0",
     "mocha": "5.2.0",
     "prettier": "~1.14.2",
     "proxyquire": "0.5.1",
-    "remark-cli": "~6.0.1",
-    "remark-preset-lint-recommended": "~3.0.2",
+    "remark-cli": "~8.0.1",
+    "remark-preset-lint-recommended": "~4.0.1",
     "rewire": "~4.0.1",
     "should": "8.2.2",
     "sinon": "~6.1.0",
     "sinon-chai": "~3.2.0",
-    "textlint": "~11.0.1",
+    "textlint": "~11.7.6",
+    "textlint-filter-rule-comments": "~1.2.2",
```

Comment:
Most of the change here seem to be package versions upgrade, except textlint-rule-no-dead-link removal (already explained in https://github.com/telefonicaid/iotagent-json/pull/515#issuecomment-736355544) and textlint-filter-rule-comments addition.

What is the purpose of textlint-filter-rule-comments module?

Reply:
[textlint-filter-rule-comments](https://github.com/textlint/textlint-filter-rule-comments) is a textlint rule that ignore error using comments directive.  Its presence allows for rules to be ignored at a finer grain.

Reply:
Thanks for the clarification. NTC.